locations/spiders/travis_perkins_gb.py

```diff
@@ -0,0 +1,71 @@
+from typing import Any, Iterable
+
+from scrapy import Request, Spider
+from scrapy.http import JsonRequest, Response
+
+from locations.dict_parser import DictParser
+from locations.hours import OpeningHours
+from locations.pipelines.address_clean_up import merge_address_lines
+
+
+class TravisPerkinsGBSpider(Spider):
+    name = "travis_perkins_gb"
+    item_attributes = {"brand": "Travis Perkins", "brand_wikidata": "Q2450664"}
+    BENCHMARX = {"brand": "Benchmarx", "brand_wikidata": "Q102181127"}
```

Comment:
https://github.com/alltheplaces/alltheplaces/blob/f7eb8baf14984461fadb5993e545019706d8b135/locations/spiders/benchmarx_gb.py#L13 - Q102181127

Reply:
https://alltheplaces-data.openaddresses.io/map.html?show=https%3A%2F%2Falltheplaces-data.openaddresses.io%2Fruns%2F2024-08-17-13-32-02%2Foutput%2Fbenchmarx_gb.geojson - 30 results, but has dropped from 128 in the last month